src/course/syllabus/application/tribute-page/project.md

```diff
@@ -0,0 +1,17 @@
+Your first project is to create a _Tribute Page_. This can be about an interest of yours or a public figure you admire. You are not required to make this about yourself or disclose anything personal. The page will be built with HTML, styled with CSS and use a bit of JavaScript.
+
+If you're new to web development, we recommend you check out the [resources](./resources.md) before you get started. You might like to spend some time learning the fundamentals of HTML and CSS before you dive in. You don't need to know a lot about JavaScript to complete this project and should be a good introduction to the language.
+
+## Requirements
+
+1. The page has a title which informs the user what the site is about
```

Comment:
Yeah, markdown will always give the correct numbers so it's easier to put `1.` for them all rather than adjusting them if you delete or move something ⚡ 

e.g.
1. do this
1. do that